src/ResourceManager/EventHub/Commands.EventHub/help/Remove-AzureRmEventHubNamespace.md

```diff
@@ -5,15 +5,22 @@ online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.event
 schema: 2.0.0
 ---
 
-# Remove-AzureRmEventHubNamespace
+# Remove-AzureRmServiceBusNamespace
```

Comment:
@v-Ajnava it looks like something was generated incorrectly in this file as this line and the parameter sets down below use `Remove-AzureRmServiceBusNamespace`